src/components/AddressSearch/index.tsx

```diff
@@ -414,10 +438,11 @@ function AddressSearch(
                         }}
                         styles={{
                             textInputContainer: [styles.flexColumn],
-                            listView: [StyleUtils.getGoogleListViewStyle(displayListViewBorder), styles.overflowAuto, styles.borderLeft, styles.borderRight, !isFocused && {height: 0}],
+                            listView: [StyleUtils.getGoogleListViewStyle(displayListViewBorder), styles.borderLeft, styles.borderRight, styles.flexGrow0, !isFocused && styles.h0],
```

Comment:
Is this change still needed?

Reply:
@eVoloshchak part of it makes sense, but `styles.flexGrow0` is not necessary, so I just removed it! Thanks 😄